src/force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,42 @@
+public with sharing class AccountTriggerHandler {
+    @TestVisible
+    public static AccountTriggerHandler instance;
+
+    public static AccountTriggerHandler getInstance() {
+        if (instance == null) {
+            instance = new AccountTriggerHandler();
+        }
+        return instance;
+    }
+
+    public void run(List<Account> newRecords, Map<Id,Account> oldRecords, TriggerOperation operation) {
+        if (!CustomSettingsSelector.getInstance().getFeaturesEnabler().Account_Trigger__c) {
+            return;
+        }
+        switch on operation {
+            when BEFORE_INSERT {
+                onBeforeInsert(newRecords);
+            }
+            when BEFORE_UPDATE {
+                onBeforeUpdate(newRecords, oldRecords);
+            }
+            when else {
+                throw new IllegalArgumentException(Constants.EXCEPTION_UNSUPPORTED_TRIGGER_OPERATION_TYPE);
+            }
+        }
+    }
+
+    @TestVisible
+    private void onBeforeInsert(List<Account> newRecords) {
+        if (CustomSettingsSelector.getInstance().getFeaturesEnabler().Account_NewRecordIsAlwaysActive__c) {
+            AccountService.getInstance().setNewAccountsAsActive(newRecords);
```

Comment:
I would suggest renaming `setNewAccountsAsActive` to `activateNewAccounts`.

5 mins to fix.